apps/site/assets/ts/stop/components/StopPageRedesign.tsx

```diff
@@ -82,12 +79,19 @@ const StopPageRedesign = ({
           </ul>
           <button type="button">Plan your Trip PLACEHOLDER</button>
         </div>
-        <div className="hidden-sm-down">
+        <div className="hidden-sm-down map-container">
           Map PLACEHOLDER Imageine a pretty map
         </div>
       </div>
       {/* Station Information Div */}
       <footer>{stop && <StationInformation stop={stop} />}</footer>
+      <div className="station-amenities">
+        <div className="station-amenity" />
+        <div className="station-amenity" />
+        <div className="station-amenity" />
+        <div className="station-amenity" />
+        <div className="station-amenity" />
+      </div>
```

Comment:
This should be in the `StationInformation` component

Reply:
Good catch - @i0sea the `StationInformation`component will include the whole bottom section, including this list of amenities, so let's put that there.